docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,425 @@
+  of the [`--cpu-quota`](https://docs.docker.com/engine/reference/run/#/cpu-quota-constraint)
+  flag in the `docker run` command.  he [`--cpu-period`] flag is set to 100000,
+   which represents the default 100ms period for measuring quota usage. The
+   kubelet enforces cpu limits if it is started with the
+  [`--cpu-cfs-quota`] flag set to true. As of Kubernetes version 1.2, this flag
+  defaults to true.
+
+- The `spec.containers[].resources.limits.memory` is converted to an integer, and
+  used as the value of the
+  [`--memory`](https://docs.docker.com/engine/reference/run/#/user-memory-constraints)
+  flag in the `docker run` command.
+
+If a Container exceeds its memory limit, it might be terminated. If it is
+restartable, the kubelet will restart it, as with any other type of runtime
+failure.
+
+If a Container exceeds its memory request, it is likely that its Pod will
+be evicted whenever the node runs out of memory.
+
+A Container might or might not be allowed to exceed its CPU limit for extended
+periods of time. However, it will not be killed for excessive CPU usage.
+
+To determine whether a Container cannot be scheduled or is being killed due to
+resource limits, see the
+[Troubleshooting](#troubleshooting) section.
+
+## Monitoring compute resource usage
+
+The resource usage of a Pod is reported as part of the Pod status.
+
+If [optional monitoring](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/README.md)
+is configured for your cluster, then Pod resource usage can be retrieved from
+the monitoring system.
+
+## Troubleshooting
+
+### My Pods are pending with event message failedScheduling
+
+If the scheduler cannot find any node where a Pod can fit, the Pod remains
+unscheduled until a place can be found. An event is produced each time the
+scheduler fails to find a place for the Pod, like this:
+
+```shell
+$ kubectl describe pod frontend | grep -A 3 Events
+Events:
+  FirstSeen LastSeen   Count  From          Subobject   PathReason      Message
+  36s   5s     6      {scheduler }              FailedScheduling  Failed for reason PodExceedsFreeCPU and possibly others
+```
+
+In the preceding example, the Pod named "frontend" fails to be scheduled due to
+insufficient CPU resource on the node. Similar error messages can also suggest
+failure due to insufficient memory (PodExceedsFreeMemory). In general, if a Pod
+is pending with a message of this type, there are several things to try:
+
+- Add more nodes to the cluster.
+- Terminate unneeded Pods to make room for pending Pods.
+- Check that the Pod is not larger than all the nodes. For example, if all the
+  nodes have a capacity of `cpu: 1`, then a Pod with a limit of `cpu: 1.1` will
+  never be scheduled.
+
+You can check node capacities and amounts allocated with the
+`kubectl describe nodes` command. For example:
+
+```shell
+$ kubectl describe nodes gke-cluster-4-386701dd-node-ww4p
```

Comment:
This needs to be updated to include `Allocatable` in addition to `Allocated`. You can use the snippet below and update the text to use the number from this snippet.

```text
$ kubectl.sh describe nodes e2e-test-minion-group-4lw4
Name:			e2e-test-minion-group-4lw4
[ ... lines removed for clarity ...]
Capacity:
 alpha.kubernetes.io/nvidia-gpu:	0
 cpu:					2
 memory:				7679792Ki
 pods:					110
Allocatable:
 alpha.kubernetes.io/nvidia-gpu:	0
 cpu:					1800m
 memory:				7474992Ki
 pods:					110
[ ... lines removed for clarity ...]
Non-terminated Pods:		(5 in total)
  Namespace			Name							CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----							------------	----------	---------------	-------------
  kube-system			fluentd-gcp-v1.38-28bv1					100m (5%)	0 (0%)		200Mi (2%)200Mi (2%)
  kube-system			kube-dns-3297075139-61lj3				260m (13%)	0 (0%)		100Mi (1%)170Mi (2%)
  kube-system			kube-proxy-e2e-test-vishnuk-minion-group-4lw4		100m (5%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			monitoring-influxdb-grafana-v4-z1m12			200m (10%)	200m (10%)	600Mi (8%)600Mi (8%)
  kube-system			node-problem-detector-v0.1-fj7m3			20m (1%)	200m (10%)	20Mi (0%)100Mi (1%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  680m (34%)	400m (20%)	920Mi (12%)	1070Mi (14%)
```

Specifically, if a pod requests more than `1120m` CPUs or `6.23Gi`, it will not fit on the node.